### ISSUE TITLE
does not run under Gnome 42 and wayland

### ISSUE THREAD

**Issue by dpriedel** [yatli/fvim#230]
I have built the latest fvim version from git using the AUR package.

When I then run fvim under Gnome 42/wayland, all I get is a blank window which does not respond to any keyboard or mouse input.  I have to force the application to quit by 'control-C' 

Fvim does run under Gnome 42/Xorg.

Is this expected?

Thanks

**Comment by yatli** [comment]
I did have it working under Arch/GNOME42/Wayland. Not sure what's going on here.

Is the blank window picking up colorscheme?

**Comment by dpriedel** [comment]
No.  The window is completely empty and does not respond to any mouse 
clicks or keyboard input.


Here is trace output


fvim --trace-to-stdout
log: fvim started. time = 8/15/2022 4:00:05 PM
model: starting neovim instance...
model: opts = Embedded ("nvim", ["--embed"], 
System.Text.UTF8Encoding+UTF8EncodingSealed)
neovim.process: Starting process. Program: nvim; Arguments: ["--embed"]
neovim.process: begin read loop
model: commencing early initialization...
editorvm #1: ctor
ui: GetTypeface: allocating new typeface Monospace:false:false
frame: configBackground: SolidBackground (1.0, #00000000)
ui: SetWindowBackgroundComposition: desired=None actual=Transparent
mainwindow: set position: 300, 300
main: mainwin bound adjusted to 300, 300, 800, 600
editor #1: viewmodel connected
editor #1: render tick 0
editor #1: MeasureOverride: 800, 600
editorvm #1: set measured size: 800, 600
ui: GetTypeface: allocating new typeface :false:false
mainwindow: first PositionChanged event: 37, 106 (delta=-263, -194)

**Comment by yatli** [comment]
looks like early initialization is stuck.
does `fvim -u NORC` help?

**Comment by dpriedel** [comment]
No.


trace looks the same, behavior still the same


fvim -u NORC --trace-to-stdout
log: fvim started. time = 8/16/2022 9:37:12 AM
model: starting neovim instance...
model: opts = Embedded
   ("nvim", ["--embed"; "-u"; "NORC"],
    System.Text.UTF8Encoding+UTF8EncodingSealed)
neovim.process: Starting process. Program: nvim; Arguments: ["--embed"; 
"-u"; "NORC"]
neovim.process: begin read loop
model: commencing early initialization...
editorvm #1: ctor
ui: GetTypeface: allocating new typeface Monospace:false:false
frame: configBackground: SolidBackground (1.0, #00000000)
ui: SetWindowBackgroundComposition: desired=None actual=Transparent
mainwindow: set position: 300, 300
main: mainwin bound adjusted to 300, 300, 800, 600
editor #1: viewmodel connected
editor #1: render tick 0
editor #1: MeasureOverride: 800, 600
editorvm #1: set measured size: 800, 600
ui: GetTypeface: allocating new typeface :false:false
mainwindow: first PositionChanged event: 960, 69 (delta=660, -231)
